### PR TITLE
Unskip CLI tests

### PR DESCRIPTION
**Pull request by Jayashsatolia403**

### CODE REVIEW

File: tests/cli_tests/0_zboxcli_file_copy_test.go [hunk 1]
```diff
@@ -21,81 +24,22 @@ func TestFileCopy(testSetup *testing.T) { // nolint:gocyclo // team preference i
...
-		destpath := "/"
-
-		allocationID := setupAllocation(t, configPath, map[string]interface{}{
-			"size": allocSize,
-		})
-
-		output, err := uploadFile(t, configPath, map[string]interface{}{
-			"allocation": allocationID,
-			"remotepath": remotePath,
-			"localpath":  file,
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/0_zboxcli_file_copy_test.go:45]: Why is this being removed?

File: tests/cli_tests/0_zboxcli_file_copy_test.go [hunk 2]
```diff
@@ -21,81 +24,22 @@ func TestFileCopy(testSetup *testing.T) { // nolint:gocyclo // team preference i
...
+	t.TestSetup("register wallet and get blobbers", func() {
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/0_zboxcli_file_copy_test.go:27]: change description

File: tests/cli_tests/0_zboxcli_file_upload_test.go [hunk 3]
```diff
@@ -24,7 +26,21 @@ func TestUpload(testSetup *testing.T) {
 	t := test.NewSystemTest(testSetup)
 	t.SetSmokeTests("Upload File With half Size of the Allocation Should Work")
 
-	t.Parallel()
+	t.TestSetup("register wallet and get blobbers", func() {
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/0_zboxcli_file_upload_test.go:29]: update description, it's not registering or getting blobbers

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]
Minor comments + lint fixes please

File: tests/cli_tests/0_zboxcli_file_copy_test.go [hunk 4]
```diff
@@ -3,7 +3,10 @@ package cli_tests
 import (
 	"encoding/json"
 	"fmt"
+	"math"
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/0_zboxcli_file_copy_test.go:6]: This file shouldn't need to be ran sequentially.

File: tests/cli_tests/0_zboxcli_file_upload_test.go [hunk 5]
```diff
@@ -7,6 +7,8 @@ import (
 	"os"
 	"path"
 	"path/filepath"
+	"regexp"
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/0_zboxcli_file_upload_test.go:10]: This file shouldn't need to be ran sequentially

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

File: tests/cli_tests/0_zboxcli_file_copy_test.go (hunk 4 above)
Comment (Jayashsatolia403) [reply tests/cli_tests/0_zboxcli_file_copy_test.go:6]: It was to debug the runner issue only. Will revert it.

File: tests/cli_tests/zboxcli_file_copy_test.go [hunk 6]
```diff
@@ -646,6 +649,69 @@ func TestFileCopy(testSetup *testing.T) { // nolint:gocyclo // team preference i
 		require.NotNil(t, err, strings.Join(output, "\n"))
 		require.Contains(t, strings.Join(output, "\n"), "Invalid path record not found")
 	})
+
+	t.RunWithTimeout("File copy - Users should be charged for copying a file ", 3*time.Minute, func(t *test.SystemTest) {
+		t.Skip("Skipping until the issue that running it in workflow is fixed")
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_file_copy_test.go:654]: It's still skipped

File: tests/cli_tests/zboxcli_file_upload_test.go [hunk 7]
```diff
@@ -715,6 +717,79 @@ func TestUpload(testSetup *testing.T) {
 		require.NotContains(t, output[0], filename)
 	})
 
+	t.RunWithTimeout("Tokens should move from write pool balance to challenge pool acc. to expected upload cost", 10*time.Minute, func(t *test.SystemTest) {
+		t.Skip("Skipping until the issue that running it in workflow is fixed")
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_file_upload_test.go:721]: It's still skipped.

File: tests/cli_tests/zboxcli_share_file_test.go [hunk 8]
```diff
@@ -1284,47 +1183,6 @@ func TestShareFile(testSetup *testing.T) {
...
-		shareParams := map[string]interface{}{
-			"allocation": allocationID,
-			"remotepath": remoteOwnerPath,
-			"clientid":   clientId,
-		}
-		output, err = shareFile(t, configPath, shareParams)
-		require.NotNil(t, err, strings.Join(output, "\n"))
-		require.Equal(t, "invalid_private_share: private sharing is only available for encrypted file", output[0], strings.Join(output, "\n"))
-		require.Len(t, output, 1, "share file - Unexpected output", strings.Join(output, "\n"))
-	})
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_share_file_test.go:1327]: Why is this removed? It's a passing test covering a negative flow.

File: tests/cli_tests/zboxcli_sync_test.go [hunk 9]
```diff
@@ -738,6 +738,313 @@ func TestSyncWithBlobbers(testSetup *testing.T) {
...
+			if item.Name == "root.txt" {
+				file = item
+			}
+		}
+		require.NotNil(t, file, "sync error, file 'root.txt' must've been uploaded to the allocation", files2)
+
+		require.Greater(t, file.Size, file_initial.Size, "file expected to be updated to bigger size")
+	})
+
+	t.Run("BROKEN Sync path to non-empty allocation - locally updated files (in sub folder) must be updated in allocation but is not see zboxcli/issues/250", func(t *test.SystemTest) {
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_sync_test.go:823]: In fixed tests, please remove all `BROKEN` references

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

File: tests/cli_tests/zboxcli_share_file_test.go (hunk 8 above)
Comment (Jayashsatolia403) [reply tests/cli_tests/zboxcli_share_file_test.go:1327]: You can't share unencrypted file in private. We had this discussion earlier. 

I have another test that covered encrypted file share.

File: tests/cli_tests/zboxcli_sync_test.go (hunk 9 above)
Comment (Jayashsatolia403) [reply tests/cli_tests/zboxcli_sync_test.go:823]: Makes sense. Let me make the changes.

**Review by Kishan-Dhakan — COMMENTED** [review]
LGTM, but do not merge till full run

**Comment by Jayashsatolia403** [comment]
Full green run here against sprint branch

**Review by Kishan-Dhakan — APPROVED** [review]
lgtm

**Pull request merged**